clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/security/DockerRegistryNamedAccountCredentials.groovy

```diff
@@ -254,6 +254,13 @@ class DockerRegistryNamedAccountCredentials implements AccountCredentials<Docker
     return this.credentials?.client?.basicAuth ?: ""
   }
 
+  @JsonIgnore
+  List<String> getTags(String repository) {
+    return credentials.client.getTags(repository).tags.sort{ tag ->
+      credentials.client.getCreationDate(repository, tag)
```

Comment:
This could cause problems with some of the public registries. I feature flagged the digest downloads because the 'get manifest' endpoint was surprisingly flaky. Could you put this sort behind that same digest/manifest feature flag?